src/main/java/de/tum/in/www1/artemis/domain/SolutionProgrammingExerciseParticipation.java

```diff
@@ -0,0 +1,100 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import javax.persistence.*;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonView;
+import de.tum.in.www1.artemis.domain.view.QuizView;
+
+@Entity
+@DiscriminatorValue(value = "SPEP")
+public class SolutionProgrammingExerciseParticipation extends Participation implements ProgrammingExerciseParticipation {
```

Comment:
I think it's good to also differentiate between the Template/Solution Participation in some instances, e.g. when creating the test cases from a Solution Participation's build result. :+1: